cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -32,7 +32,6 @@ import (
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
 	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
-	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
i think there was a plan to enable a tool to catch double imports, but it hasn't made it in k/k yet.

Reply:
I want to know ,if I found other double imports,which one should I leave?
Just like the above, should I leave 34 line or 35 line?

Reply:
my guess is that across the kubeadm code base we are more consistent using `kubeadmconstants`. this PR moved that in the other direction, but overall as long as the imports don't have conflicting (in meaning) names and if there are no double imports, it's OK.

Reply:
I understand, thank you very much.